dkms/DIFFERENCES.md

```
@@ -2,23 +2,72 @@
-## Patches from Mainline Patchsets
+## Upstream Patch Series (v7)
 
-The following patches are from our patchsets submitted to linux-wireless:
-- See `kernels/nbd168/*.patch` for the full list (22 patches)
-- These fix NULL pointer dereferences, mutex issues, MLO bugs, etc.
-- Tracking: Submitted as v4 patchset to linux-wireless mailing list
+The following 6 patches have been submitted to nbd168 (Felix Fietkau) for upstream:
 
-## Additional Patches (DKMS Only)
+| # | Patch | Status |
+|---|-------|--------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)